src/core/annotation.js

```diff
@@ -68,10 +68,12 @@ AnnotationFactory.prototype = /** @lends AnnotationFactory.prototype */ {
    * @param {Object} ref
    * @param {string} uniquePrefix
    * @param {Object} idCounters
+   * @param {boolean} renderInteractiveForms
    * @returns {Annotation}
    */
-  create: function AnnotationFactory_create(xref, ref, uniquePrefix,
-                                            idCounters) {
+  create:
+      function AnnotationFactory_create(xref, ref, uniquePrefix, idCounters,
+                                        renderInteractiveForms) {
```

Comment:
Nit: I think that you could just do this instead (note the previous comment about changing the `blame`):
```javascript
  create: function AnnotationFactory_create(xref, ref,
                                            uniquePrefix, idCounters,
                                            renderInteractiveForms) {
```